godotenv.go

```
@@ -308,6 +318,16 @@ func parseValue(value string, envMap map[string]string) string {
 		}
 	}
 
+	// embed commands
+	if shouldParseEmbed {
+		if strings.HasPrefix(value, "$(") && strings.HasSuffix(value, ")") {
+			cmdStr := value[2 : len(value)-1]
+			cmd := exec.Command("/bin/sh", "-c", cmdStr)
+			out, _ := cmd.Output()
```

Why silence the error here? Wouldn't that make it harder to diagnose failures?